curriculum/challenges/english/01-responsive-web-design/basic-css/use-hex-code-for-specific-colors.md

```diff
@@ -11,7 +11,7 @@ dashedName: use-hex-code-for-specific-colors
 
 Did you know there are other ways to represent colors in CSS? One of these ways is called hexadecimal code, or hex code for short.
 
-We usually use <dfn>decimals</dfn>, or base 10 numbers, which use the symbols 0 to 9 for each digit. <dfn>Hexadecimals</dfn> (or <dfn>hex</dfn>) are base 16 numbers. This means it uses sixteen distinct symbols. Like decimals, the symbols 0-9 represent the values zero to nine. Then A,B,C,D,E,F represent the values ten to fifteen. Altogether, 0 to F can represent a digit in hexadecimal, giving us 16 total possible values. You can find more information about [hexadecimal numbers here](https://en.wikipedia.org/wiki/Hexadecimal).
+We usually use <dfn>decimals</dfn>, or base 10 numbers, which use the symbols 0 to 9 for each digit. <dfn>Hexadecimals</dfn> (or <dfn>hex</dfn>) are base 16 numbers. This means it uses sixteen distinct symbols. Like decimals, the symbols 0-9 represent the values zero to nine. Then A,B,C,D,E,F represent the values ten to fifteen. Altogether, 0 to F can represent a digit in hexadecimal, giving us 16 total possible values. You can find more information about [hexadecimal numbers here](https://www.freecodecamp.org/news/hexadecimal-number-system/).
 
 In CSS, we can use 6 hexadecimal digits to represent colors, two each for the red (R), green (G), and blue (B) components. For example, `#000000` is black and is also the lowest possible value. You can find more information about the [RGB color system here](https://en.wikipedia.org/wiki/RGB_color_model).
```

Comment:
Perhaps this link also could be changed - to the https://www.freecodecamp.org/news/rgb-color-html-and-css-guide/#whatisthergbcolormodel?

Reply:
it could work but I can't find the right id to use there, it always just bring to the top of the article

Reply:
Adding #whatisthergbcolormodel worked for me

Reply:
it wasn't working for me 🤨
weird, I will change it